### ISSUE TITLE
Update to 0.5.0 breaks magic-abbrevations

### ISSUE THREAD

**Issue by open-dynaMIX** [zsh-users/zsh-autosuggestions#404]
First of all: Thanks a lot for creating `zsh-autosuggestions`!

After upgrading to 0.5.0, magic-abbrevations stopped working. Downgrading to 0.4.3 fixes the issue.

OS: Arch Linux
ZSH: zsh 5.6.2 (x86_64-pc-linux-gnu)

Minimal `.zshrc` that causes the problem:

```
setopt extendedglob

. /usr/share/zsh/plugins/zsh-autosuggestions/zsh-autosuggestions.zsh

# Magic
set_abbrevations() {
    typeset -Ag abbreviations
    abbreviations=(
      "Le"    "| less"
      "Eg"    "| egrep"
      "Ta"    "| tail"
      "Tl"    "tail -F"
      "So"    "| sort"
    )
}

magic-abbrev-expand() {
    set_abbrevations
    local MATCH
    LBUFFER=${LBUFFER%%(#m)[_a-zA-Z0-9]#}
    command=${abbreviations[$MATCH]}
    LBUFFER+=${command:-$MATCH}

    if [[ "${command}" =~ "__CURSOR__" ]]
    then
        RBUFFER=${LBUFFER[(ws:__CURSOR__:)2]}
        LBUFFER=${LBUFFER[(ws:__CURSOR__:)1]}
    else
        zle self-insert
    fi
}

no-magic-abbrev-expand() {
    LBUFFER+=' '
}

zle -N magic-abbrev-expand
zle -N no-magic-abbrev-expand
bindkey " " magic-abbrev-expand
bindkey "^x " no-magic-abbrev-expand
bindkey -M isearch " " self-insert

```

My whole config can be found in my [dotfiles](https://github.com/open-dynaMIX/dotfiles/tree/master/dotfiles/.zshrc.d).

Thanks for your help!

**Comment by bkbooth** [comment]
I think this has been fixed in #379

It's been merged into `develop`, just waiting on a release
